frontend/src/app/features/team-planner/team-planner/planner/team-planner.component.html

```diff
@@ -56,18 +56,46 @@
     ></op-tp-add-assignee>
   </ng-template>
 
-  <div
-    class="wp-inline-create-button op-team-planner--add-assignee"
-    *ngIf="!(showAddAssignee$ | async)"
-  >
-    <button
-      type="button"
-      class="wp-inline-create--add-link tp-assignee-add-button"
-      (click)="showAssigneeAddRow()"
-      data-qa-selector="tp-assignee-add-button"
-    >
-      <op-icon icon-classes="icon-context icon-add"></op-icon>
-      <span [textContent]="text.add_assignee"></span>
-    </button>
+  <div class="op-team-planner--footer" data-qa-selector="op-team-planner-footer">
+    <div class="op-team-planner--add-assignee">
+      <div
+        class="wp-inline-create-button"
+        *ngIf="!(showAddAssignee$ | async)"
+      >
+        <button
+          type="button"
+          class="wp-inline-create--add-link tp-assignee-add-button"
+          (click)="showAssigneeAddRow()"
+          data-qa-selector="tp-assignee-add-button"
+        >
+          <op-icon icon-classes="icon-context icon-add"></op-icon>
+          <span [textContent]="text.add_assignee"></span>
+        </button>
+      </div>
+    </div>
+    <ng-container *ngIf="(dropzone$ | async) as dropzone">
+      <div
+        *ngIf="dropzone.dragging"
+        #removeDropzone
+        (mouseenter)="dropzoneHovered$.next(true)"
+        (mouseleave)="dropzoneHovered$.next(false)"
+        (mouseup)="dropzone.canDrop && removeEvent(dropzone.dragging)"
+        class="op-team-planner--remove-dropzone"
+        data-qa-selector="op-team-planner-dropzone"
+        [class.op-team-planner--remove-dropzone_active]="dropzone.isHovering && dropzone.canDrop"
+        [class.op-team-planner--remove-dropzone_forbidden]="!dropzone.canDrop"
+      >
+      <span
+        *ngIf="dropzone.canDrop"
+        class="op-team-planner--dropzone-label"
+        [textContent]="text.drag_here_to_remove"
+      ></span>
+        <span
+          *ngIf="!dropzone.canDrop"
+          class="op-team-planner--dropzone-label op-team-planner--dropzone-label_forbidden"
```

Comment:
I don't think you need the extra forbidden class here if it's on the parent already